bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -291,16 +292,17 @@ Use the `SCRIPT` transformation with JavaScript Scripting by:
 Full documentation for the openHAB JavaScript library can be found at [openhab-js](https://openhab.github.io/openhab-js).
 
 The openHAB JavaScript library provides type definitions for most of its APIs to enable code completion is IDEs like [VS Code](https://code.visualstudio.com).
-To use the type definitions, install the [`openhab` npm package](https://npmjs.com/openhab) (read the [installation guide](https://github.com/openhab/openhab-js#custom-installation) for more information).
-If an API does not provide type definitions and therefore autocompletion won‘t work, the documentation will include a note.
+To use the type definitions, install the [`openhab` npm package](https://npmjs.com/openhab) (read the [installation guide](https://github.com/openhab/openhab-js#custom-installation) for more information), and import the used namespaces with `const { rules, triggers, items } = require('openhab');` (adjust this to your needs).
+If an API does not provide type definitions and therefore autocompletion won't work, the documentation will include a note.
 
 ### Items
 
-The items namespace allows interactions with openHAB items.
+The Items namespace allows interactions with openHAB items.
```

Comment:
It seems to be in lowercase below? Perhaps:
```suggestion
The `items` namespace allows interactions with openHAB items.
```

?

Reply:
Thanks for the catch. I first have to correct this on the openhab-js repo so I don’t revert that the next time I update the addon README.

Reply:
I will probably add some doc updates to this PR coming from openhab-js.

Reply:
Done.